chromium_src/components/os_crypt/os_crypt_win.cc

```diff
@@ -1,8 +1,16 @@
 #include "base/command_line.h"
+#include "brave/common/brave_switches.h"
+
+// switches::kDisableEncryption
+const char kDisableEncryption[] = "disable-encryption";

```

Comment:
just thinking now that we should probably make this `disable-encryption-win` just to be clear that it's windows-only. I think the machine-id is win-only anyway, right?

Reply:
Yes exactly

Reply:
@crazy-max yes, but currently it is only windows so I feel like it could be misleading

Reply:
Ok done :)